examples/magento-graphcms/pages/account/signin.tsx

```diff
@@ -20,7 +20,11 @@ function AccountSignInPage() {
 
   return (
     <>
-      <PageMeta title={i18n._(/* i18n */ 'Sign in')} metaRobots={['noindex']} />
+      <PageMeta
+        title={i18n._(/* i18n */ 'Sign in')}
+        metaDescription={i18n._(/* i18n */ 'Sign in')}
```

Comment:
Page is not indexable, metaDescription is useless. Adding this change creates useless churn